modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/lbs_discrete_ordinates_solver.cc

```diff
@@ -480,7 +478,110 @@ DiscreteOrdinatesSolver::SetPrimarySTLvectorFromMultiGSPETScVecFrom(const std::v
     }
   } // for groupset id
 
-  VecRestoreArrayRead(x_src, &x_ref);
+  VecRestoreArrayRead(x, &x_ref);
+}
+
+void
+DiscreteOrdinatesSolver::ReorientAdjointSolution()
+{
+  for (const auto& groupset : groupsets_)
+  {
+    int gs = groupset.id_;
+
+    // Moment map for flux moments
+    const auto& moment_map = groupset.quadrature_->GetMomentToHarmonicsIndexMap();
+
+    // Angular flux info
+    auto& psi = psi_new_local_[gs];
+    const auto& uk_man = groupset.psi_uk_man_;
+
+    // Build reversed angle mapping
+    std::map<int, int> reversed_angle_map;
+    if (options_.save_angular_flux)
+    {
+      const auto& omegas = groupset.quadrature_->omegas_;
+      const auto num_gs_angles = omegas.size();
+
+      // Go through angles until all are paired
```

Comment:
angles --> directions

Reply:
This is a less trivial change. Most of the code base calls directions angles, not the other way around... See `modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep_chunks/aah_sweep_chunk.cc` for example. Thoughts?

Reply:
This comment was specific to the commented line:

// Go through angles until all are paired
-->
// Go through directions until all are paired

Reply:
if you are referring to angle_sets, I am willing to discuss this elsewhere. 
Here, we are really talking about psi^*(omega) = psi(-omega) and I think we would more likely say [gerund verb] the opposite direction rather than [gerund] the opposite angle.

Reply:
I am referring to the 24 instances of "angle" throughout (the sweeper loops over angles, not directions) versus the 7 instances of "direction" within as many lines of code. Using "direction" is the exception, not the rule in the code base, which is why I am hesitant to break from the existing convention. And yes, I am aware that these are all local variables. 

I tend to agree with you that maybe direction is better than angle, but I don't think we should start deciding to break from existing convention in small parts of the code via PRs separate topics. Let's create an issue and discuss further.